pkg/kotsadm/proxy.go

```diff
@@ -0,0 +1,34 @@
+package kotsadm
+
+import (
+	"github.com/replicatedhq/kots/pkg/kotsadm/types"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func getProxyEnv(deployOptions types.DeployOptions) []corev1.EnvVar {
+	result := []corev1.EnvVar{
+		{
+			Name:  "HTTP_PROXY",
+			Value: deployOptions.HTTPProxyEnvValue,
+		},
+		{
+			Name:  "HTTPS_PROXY",
+			Value: deployOptions.HTTPSProxyEnvValue,
+		},
+	}
+
+	kotsadmNoProxy := "kotsadm-postgres,kotsadm-minio,kotsadm-api-node"
```

Comment:
Do we need this on the kotsadm service also?

Reply:
@marccampbell only operator connects to that and it has no proxy defined, so it's always a direct connection.